src/renderer/utils/Monitor.tsx

```diff
@@ -0,0 +1,13 @@
+import { Subject } from 'rxjs';
+
+export interface Monitor {
+  watch(events: Event[]): void;
+  unwatch(events: string[]): void;
+  ignoreFiles(eventToIgnore: Event[], ignoredFilepath: string[]): void;
+  closeAll(): void;
+  getEvents(): Event[];
+  getEventsWatching(): Event[];
+  getSubject(): Subject<[Event, string]>;
+}
+
+export type Event = 'unlink' | 'unlinkDir' | 'add' | 'addDir' | 'change' | 'error';
```

Comment:
It is better to make a more detailed input argument since user might need to watch only files added not the directory added, or the other way around.